docs/guides/continuous-integration/github-actions.mdx

```diff
@@ -353,7 +353,7 @@ jobs:
 
   ui-chrome-tests:
     runs-on: ubuntu-22.04
-    container: cypress/browsers:node12.18.3-chrome87-ff82
+    container: cypress/browsers:latest
```

Comment:
See also changes proposed in PR https://github.com/cypress-io/cypress-documentation/pull/5212

Reply:
Updated to match node 16 and chrome 107